zebra/zebra_evpn_mac.c

```diff
@@ -2108,14 +1997,12 @@ int zebra_evpn_mac_remote_macip_add(struct zebra_evpn *zevpn,
 					zevpn->vni, macaddr,
 					mac->es ? mac->es->esi_str : "-",
 					mac->loc_seq,
-					zebra_evpn_zebra_mac_flag_dump(
-						mac, mac_buf, sizeof(mac_buf)));
+					zebra_evpn_zebra_mac_flag_dump(mac, mac_buf, sizeof(mac_buf)));
 			}
 
 			zebra_evpn_mac_clear_sync_info(mac);
-			zebra_evpn_mac_send_del_to_client(zevpn->vni, macaddr,
-							  mac->flags,
-							  false /* force */);
+			zebra_evpn_mac_send_del_to_client(zevpn->vni, macaddr, mac->flags,
+										false /* force */);
```

Comment:
some of these changes like this one appear not to match the style/checkpatch rules - is that just an artifact of the github web ui? do these pass the normal style checks?

Reply:
Thanks @mjstapp for your time.
In github web ui the alignment is looking like that, and the style checks are passed.
Infact am trying to correct these kind of issue as our team is taking more time for code review/understanding due to the existing alignments.